From fdfff7cf6013a030f2b240b13b02831e502d49e7 Mon Sep 17 00:00:00 2001
From: Andrew Eikum <aeikum@codeweavers.com>
Date: Wed, 21 Aug 2013 10:14:59 -0500
Subject: [PATCH] winmm: Increase mmdevapi buffer sizes.

The tiny buffer size caused audio capture glitches on OSX, where the OS
often returns audio in half-second chunks.

EDIT: I'm actually reverting Andrew's patch of OSX, in my build but leaving his logic for patch / description intact - for future reference to myself. (ninez).

---
 dlls/winmm/waveform.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/dlls/winmm/waveform.c b/dlls/winmm/waveform.c
index 97849ff..23adfc0 100644
--- a/dlls/winmm/waveform.c
+++ b/dlls/winmm/waveform.c
@@ -68,8 +68,8 @@
  *   - We must be able to identify bad devices without crashing.
  */
 
-/* buffer size = 100 * 100000 (100 ns) = 1 second */
-#define AC_BUFLEN (100 * 100000)
+/* buffer size = 100 * 100000 (100 ns) = 0.1 second */
+#define AC_BUFLEN (10 * 100000)
 #define MAX_DEVICES 256
 #define MAPPER_INDEX 0x3F
 
